examples/hubot-example/script.js

```diff
@@ -145,7 +145,7 @@ app.event('member_left_channel', async ({ say }) => { await say(randomLeaveReply
   //   res.send('Missing HUBOT_ANSWER_TO_THE_ULTIMATE_QUESTION_OF_LIFE_THE_UNIVERSE_AND_EVERYTHING in environment: please set and try again')
   // })
 
-// If you ask what i the the answer to the ulimate question of life it will resond with what is in your .env file
+// If you ask what is the answer to the ultimate question of life it will respond with what is in your .env file
```

Comment:
Sometimes things like this are meant to be cute. I don't think any of these items count even if the spirit itself is...